<launch>

  <node pkg="q_learning_project" type="main.py" name="main_1"/>

  <node pkg="q_learning_project" type="color_handling.py" name="color_1"/>

  <node pkg="q_learning_project" type="robot_movement.py" name="movement_1"/>
  
</launch>
